libvast/src/system/spawn_disk_monitor.cpp

```diff
@@ -27,24 +27,36 @@ spawn_disk_monitor(node_actor::stateful_pointer<node_state> self,
   if (!archive)
     return caf::make_error(ec::missing_component, "archive");
   auto opts = args.inv.options;
+  std::optional<std::string> command;
+  if (auto cmd = caf::get_if<std::string>(&opts, "vast.start.disk-budget-check-"
+                                                 "binary"))
```

Comment:
```suggestion
  if (auto cmd = caf::get_if<std::string>(&opts, //
        "vast.start.disk-budget-check-binary"))
```

If you insert `//` at the end you force a line break so we can grep for the option.

Reply:
Maybe you have a different clang-format version than me?

```
  if (auto cmd = caf::get_if<std::string>(&opts, //
                                          "vast.start.disk-budget-check-"
                                          "binary"))
```

Reply:
I was using the default config, my bad. You can still fix this by breaking manually before `&opts`.